xilem_core/src/sequence.rs

```diff
@@ -33,7 +33,7 @@ impl<T> AppendVec<T> {
     pub fn push(&mut self, item: T) {
         self.inner.push(item);
     }
-    /// [Drain](Vec::drain) all items from this `AppendVec`.
+    /// [`Drain`](Vec::drain) all items from this `AppendVec`.
```

Comment:
This was intentionally non-graved - it's using the verb "to drain" as used in the function `drain`, rather than a reference to the function `drain` itself.

Reply:
Fixed.